README.md

```diff
@@ -44,54 +62,126 @@ This goes both ways: when explaining something make sure to present enough conte
+
+- **Sprint Planning**: Individual teams hold sprint planning meetings at the start of the sprint. Work is estimated, tickets are assigned, and the team determines how much they can commit to completing. Everyone is expected to review potential upcoming work prior to sprint planning in order to estimate and discuss the work effectively.  
+- **Mega-groom**: The full team meets to discuss work for the upcoming sprint. This is scheduled the week before sprint planning. The meeting has an agenda that is both created by both product managers and tech leads. In addition to providing visibility into upcoming work cross-team, approach and open questions are discussed at a high level. Everyone should review this document prior to the meeting so they can come prepared to discuss the work.
+- **Sprint Demos**: Near the end of the sprint on Thursday afternoons, members volunteer to demo what they've worked on. This meeting involves the full team and optionally other department members. Demos should not be overly long or technical as the audience of demos involves non-technical folks. If a deep dive or longer technical demo is desired, we can use the following week's Dev Roundtable (see below about Dev Roundtables) slot.
+- **Sprint Retro**: Immediately following demos, we meet to discuss What Went Well, What to To Improve, and Other Discussion Topics. We hold an all-teams retro, which is great for general culture, surfacing larger questions or issues, and providing full team updates. We also typically celebrate a person who’s been a Goddamn Champion (GDC) in having a positive impact through their work over the past sprint. The GDC winner receives a $25 gift card.
+
+_**Another regular meeting to note:** We typically have a brown bag lunch meeting on Wednesdays at noon. It is called Dev Roundtable. These meetings are typically technical and can involve technical demos/presentations, working sessions, or discussions._
+
+Sprints should not be booked more than **75% full**. This gives buffer for sick time, unplanned work, work that was underestimated at the start of the sprint, and breathing room for individual professional development. Additionally, a minimum 3 points of sustaining engineering work should be tackled every sprint. If a team is experiencing sprint rollover, they should step back, and re-evaluate their sprint planning strategy. Sprint rollover should be the exception not the rule.
+
+Additionally, we record best practices as it relates to groom and sprint planning in our [Sprint learnings document](https://docs.google.com/document/d/1SiRF0R73Tg0IBHHCzX5PWAfyXF_Q9TUogxxOyxR-pAM/edit#heading=h.r7t77g3jnbng).
+
+### Ticket Management
+
+All developers are expected to keep their ticket statuses up to date at all times. Just as our communication section details, no one should have to waste time figuring out the state of work. PRs should be linked to the ticket, and questions and comments should be added to the ticket as well. 
```

Comment:
Not sure if this is the correct place for it but one thing that came up recently on our team was the handling of urgent bugs. I was told that it's practice to create a separate slack incident thread or a slack standup thread for communication of status in addition to keeping the pivotal ticket up to date.  If this is already addressed somewhere else, ignore this comment!

Reply:
This isn't a written practice, but I like your idea! I'm going to add it in

Reply:
Oh, I should be clearer -- we do have in our incident playbook to create a slack thread / add it to the ticket. Would urgent bugs fall under that....thinking 🤔

Reply:
I think I'll add a note here about our Incident playbook in reference to incidents so that it's not only noted the testing and monitoring section

Reply:
Yeah, I don't think we've ever formally decided on how we handle those but I really like the idea (and have occasionally created a "manual" empty incident in PagerDuty just as a way to kick off the Slack channel).